### PR TITLE
feat: new default dashboard

### PR DESCRIPTION
**Pull request by rcmarron**
## Problem

When a new project is created, the primary dashboard (seen on the homepage) doesn't get set.

Also, the default insights + dashboard are a bit dated.

## Changes

* Set the `primary_dashboard` for the team when the team is created
* Update the default insights + dashboard to align with [this comment](https://github.com/PostHog/posthog/issues/8595#issuecomment-1058051381)

Images of new dashboard:
image
image
image

👉 *Stay up-to-date with our [coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

## How did you test this code?
Added tests to make sure the primary dashboard is set on project creation

Manually verified that the dashboard + insights are created and their layout works as the dashboard is resized.

### CODE REVIEW

**Comment by rcmarron** [comment]
@marcushyett-ph I'd love your eyes on the copy on the new default insights here. Also, 2 deviations from your recommendations:
* You had "acquisition source", which is a bit confusing because it would be the acquisition source for users who visited the site in the last 14 day (even if they were acquired well before that). Instead, I did "Referring domain"
* For your funnel of "Pageview -> Autocapture", I felt like this funnel didn't do a great job of showing an example funnel or being useful immediately. So instead, I updated the existing sample funnel to have the first step be a "Page click". The idea being that this should show some drop off and represent a better "sample funnel"

File: posthog/helpers/dashboard_templates.py [hunk 1]
```diff
@@ -107,41 +167,15 @@ def _create_default_app_items(dashboard: Dashboard) -> None:
             INSIGHT: "FUNNELS",
         },
         last_refresh=now(),
+        color="green",
+        order=5,
         is_sample=True,
         description="This is a sample of how a user funnel could look like. It represents the number of users that performed "
```
Comment (EDsCODE) [inline posthog/helpers/dashboard_templates.py:173]: The sample here after showing several graphs of usable information could be jarring/misleading. My thought process would be that the "sample" would make me rethink if the previous graphs were also sample data

**Review by EDsCODE — APPROVED** [review]
minor UX nit otherwise much better starter visualizaitons

File: posthog/helpers/dashboard_templates.py (hunk 1 above)
Comment (rcmarron) [reply posthog/helpers/dashboard_templates.py:173]: Ah yeah, that's a really good point. It's tough to think of a useful funnel that we can make on the default data...

I'm wondering if funnels shouldn't be here? Or we could just do a `Pageview -> Pageview -> Pageview` funnel?

@marcushyett-ph curious to get your thoughts on this one.

Comment (mariusandra) [reply posthog/helpers/dashboard_templates.py:173]: I think we want to encourage people to dig into their data and start exploring. `Pageview -> Pageview -> Pageview` is a funny funnel, but great canvas to start exploring from. But only in "top to bottom" mode. Left to right is broken :(.

The knowledge that [only 45% of visitors do more than 2 pageviews](https://app.posthog.com/insights/9TB833eC) is already interesting enough to get me to start asking questions. Seeing the same [broken down by browser](https://app.posthog.com/insights/mZur6WXH) could be even interesting and get me asking even more questions.

**Review by mariusandra — APPROVED** [review]

File: posthog/helpers/dashboard_templates.py (hunk 1 above)
Comment (rcmarron) [reply posthog/helpers/dashboard_templates.py]: Agreed - just updated it to `Pageview -> Pageview -> Pageview` broken down by browser.

Super easy to iterate here if we have better ideas too.

**Comment by marcushyett-ph** [comment]
@rcmarron feedback makes sense. copy looks good to me.

**Pull request merged**